src/router.js

```diff
@@ -60,20 +60,18 @@ export default Router
 
 // check if feature is allowed and redirect if not
 async function featureAvailable(store, networkSlug, feature) {
-  const networks = store.state.connection.networks
+  let networks = store.state.connection.networks
+  if (networks.length === 0) {
+    await new Promise((resolve) => setTimeout(resolve, 100))
+    featureAvailable(store, networkSlug, feature)
```

Comment:
```suggestion
    return featureAvailable(store, networkSlug, feature)
```

Reply:
what is the difference?

Reply:
I mean, I tested and it works

Reply:
without this the execution of the async function ` featureAvailable(store, networkSlug, feature)` doesn't really matter

Reply:
Ok ok, I change it

Reply:
Done